app/models/company_certification.py

```diff
@@ -11,7 +11,7 @@ class CompanyCertification(BaseModel):
     company = db.relationship("Company", backref=backref("certifications"))
 
     attribution_date = db.Column(db.Date, nullable=False)
-    expiration_date = db.Column(db.Date, nullable=True)
+    expiration_date = db.Column(db.Date, nullable=False)
```

Comment:
:orange_circle: A discuter pour voir si c'était à Nullable pour une raison métier

Reply:
`expiration_date` is not null only if the company is certified